types/pdfmake/interfaces.d.ts

```diff
@@ -0,0 +1,473 @@
+/// <reference types="node" />
+/// <reference types="pdfkit" />
+
+export type PageSize = PredefinedPageSize | CustomPageSize;
+
+export interface CustomPageSize {
+    width: number;
+    height: number | 'auto';
+}
+
+export type PredefinedPageSize =
+    | '4A0'
+    | '2A0'
+    | 'A0'
+    | 'A1'
+    | 'A2'
+    | 'A3'
+    | 'A4'
+    | 'A5'
+    | 'A6'
+    | 'A7'
+    | 'A8'
+    | 'A9'
+    | 'A10'
+    | 'B0'
+    | 'B1'
+    | 'B2'
+    | 'B3'
+    | 'B4'
+    | 'B5'
+    | 'B6'
+    | 'B7'
+    | 'B8'
+    | 'B9'
+    | 'B10'
+    | 'C0'
+    | 'C1'
+    | 'C2'
+    | 'C3'
+    | 'C4'
+    | 'C5'
+    | 'C6'
+    | 'C7'
+    | 'C8'
+    | 'C9'
+    | 'C10'
+    | 'RA1'
+    | 'RA2'
+    | 'RA3'
+    | 'RA4'
+    | 'SRA1'
+    | 'SRA2'
+    | 'SRA3'
+    | 'SRA4'
+    | 'EXECUTIVE'
+    | 'FOLIO'
+    | 'LEGAL'
+    | 'LETTER'
+    | 'TABLOID';
+
+export type PageOrientation = 'portrait' | 'landscape';
+
+export type Size =
+    | number // absolute
+    | 'auto'
+    | '*'
+    | string; // percentage
+
+export interface TFontDictionnary {
+    [fontName: string]: TFontFamilyTypes;
+}
+
+export interface TFontFamilyTypes {
+    normal?: string;
+    bold?: string;
+    italics?: string;
+    bolditalics?: string;
+}
+
+export interface TDocumentInformation {
```

Comment:
Please add back type annotation comments.

Reply:
I didn't pay attention while merging, will do.

Reply:
This looks good.